projects/components/src/table/renderers/header-cell/table-header-cell-renderer.component.ts

```diff
@@ -16,11 +17,28 @@ import { TableCellRendererService } from '../table-cell-renderer.service';
       [htcTooltip]="this.columnConfig.title"
       class="table-header-cell-renderer"
     >
-      {{ this.columnConfig.title }}
+      <htc-in-filter-button
+        class="filter-button"
+        *ngIf="this.columnConfig?.filterAttribute && this.leftAlignFilterButton"
+        [metadata]="this.metadata"
+        [attribute]="this.columnConfig.filterAttribute"
+        [values]="this.values"
+      ></htc-in-filter-button>
+      <div class="title" [ngClass]="this.classes" (click)="this.sortChange.emit()">{{ this.columnConfig.title }}</div>
+      <htc-in-filter-button
```

Comment:
Template!

Example from the data cell change since I asked there too:
```diff
diff --git a/projects/components/src/table/renderers/data-cell/table-data-cell-renderer.component.ts b/projects/components/src/table/renderers/data-cell/table-data-cell-renderer.component.ts
index 56b3ec7..e6dd6c2 100644
--- a/projects/components/src/table/renderers/data-cell/table-data-cell-renderer.component.ts
+++ b/projects/components/src/table/renderers/data-cell/table-data-cell-renderer.component.ts
@@ -21,26 +21,25 @@ import { TableCellRendererService } from '../table-cell-renderer.service';
   changeDetection: ChangeDetectionStrategy.OnPush,
   template: `
     <div class="table-data-cell-renderer" [class.selected]="this.popoverOpen">
-      <htc-filter-button
-        class="filter-button"
-        *ngIf="this.columnConfig?.filterAttribute && this.leftAlignFilterButton"
-        [metadata]="this.metadata"
-        [attribute]="this.columnConfig?.filterAttribute"
-        [value]="this.parseValue()"
-        (popoverOpen)="this.popoverOpen = $event"
-      ></htc-filter-button>
+      <ng-container *ngIf="this.columnConfig?.filterAttribute && this.leftAlignFilterButton">
+        <ng-container *ngTemplateOutlet="filterButton"></ng-container>
+      </ng-container>
       <div class="cell-renderer-content" [ngClass]="this.alignment" (click)="this.onClick()">
         <ng-container #cellRenderer></ng-container>
       </div>
+      <ng-container *ngIf="this.columnConfig?.filterAttribute && !this.leftAlignFilterButton">
+        <ng-container *ngTemplateOutlet="filterButton"></ng-container>
+      </ng-container>
+    </div>
+    <ng-template #filterButton>
       <htc-filter-button
         class="filter-button"
-        *ngIf="this.columnConfig?.filterAttribute && !this.leftAlignFilterButton"
         [metadata]="this.metadata"
         [attribute]="this.columnConfig?.filterAttribute"
         [value]="this.parseValue()"
         (popoverOpen)="this.popoverOpen = $event"
       ></htc-filter-button>
-    </div>
+    </ng-template>
   `
 })
 export class TableDataCellRendererComponent implements OnInit {

```